docs/DeveloperGuide.adoc

```diff
@@ -18,8 +18,31 @@ endif::[]
 By: `Team W16-2`      Since: `Sept 2018`      Licence: `MIT`
 
 == Introduction
-TBC
 
+Welcome to SocialCare's Developer Guide!
+
+=== What is SocialCare?
+
+SocialCare is a CLI-based event and volunteer management system designed for social welfare organisations. It enables the following:
+
+* Faster volunteer and event management (than a typical mouse/GUI driven app)
+* Tagging to categorize volunteers and events
+* Additional fields for volunteers, such as number of volunteer hours
```

Comment:
Additional field for volunteers? 

Maybe it would be better to write:
"Managing volunteer records, such as number of hours by volunteer"

Reply:
This part was adapted from README.adoc, so let's decide on an update and make amendments to both parts. I would suggest:

"Managing additional volunteer data, such as number of service hours per volunteer"

Reply:
Managing additional volunteer sounds better.

Reply:
Probably need to update the README.adoc to reflect that change too.